### PR TITLE
doc: reef 18.2.4 release note

### PR DESCRIPTION
**Pull request by yuriw**
## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "quincy"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

- When filling out the below checklist, you may click boxes directly in the GitHub web UI.  When entering or editing the entire PR message in the GitHub web UI editor, you may also select a checklist item by adding an `x` between the brackets: `[x]`.  Spaces and capitalization matter when checking off items this way.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [ ] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [ ] No impact that needs to be tracked
- Documentation (select at least one)
  - [ ] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [ ] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
- `jenkins test rook e2e`
</details>

### CODE REVIEW

**Comment by yuriw** [comment]
this has to be closed

File: doc/releases/reef.rst [hunk 1]
```diff
@@ -6,6 +6,438 @@ Reef is the 18th stable release of Ceph. It is named after the reef squid
...
+v18.2.4 Reef
+============
+
+This is the fourth backport release in the Reef series. We recommend that all users update to this release.
+
+An early build of this release was accidentally exposed and packaged as 18.2.3 by the Debian project in April.
+That 18.2.3 release should not be used. The official release was re-tagged as v18.2.4 to avoid
+further confusion.
+
+v18.2.4 container images, now based on CentOS 9, may be incompatible on older kernels (e.g., Ubuntu 18.04) due
```
Comment (anthonyeleven) [inline doc/releases/reef.rst:18]: incompatible *with* older kernels

Also, Ubuntu 18.04 isn't a kernel.

(e.g., the 4.15 kernel shipped with Ubuntu 18.04)

Ideally we would identify the threshold kernel version.

File: doc/releases/reef.rst [hunk 2]
```diff
@@ -6,6 +6,438 @@ Reef is the 18th stable release of Ceph. It is named after the reef squid
...
+may encounter crashes during pthread_create. For workarounds, refer to the related tracker. However, we recommend
+upgrading your OS to avoid this unsupported combination.
+Related tracker: https://tracker.ceph.com/issues/66989
+
+Notable Changes
+---------------
+
+* RADOS: This release fixes a bug (https://tracker.ceph.com/issues/61948) where pre-reef clients were allowed
+  to connect to the `pg-upmap-primary` (https://docs.ceph.com/en/reef/rados/operations/read-balancer/)
+  interface despite users having set `require-min-compat-client=reef`, leading to an assert in the osds
```
Comment (anthonyeleven) [inline doc/releases/reef.rst:29]: assert in OSDs

File: doc/releases/reef.rst [hunk 3]
```diff
@@ -6,6 +6,438 @@ Reef is the 18th stable release of Ceph. It is named after the reef squid
...
+  and mons. You are susceptible to this bug in reef versions prior to 18.2.4 if 1) you are using an osdmap
```
Comment (anthonyeleven) [inline doc/releases/reef.rst:30]: Reef

**Review by anthonyeleven — COMMENTED** [review]
Changes requested

**Comment by yuriw** [comment]
no need

**Pull request closed without merging**